entity-service/src/test/kotlin/com/egm/stellio/entity/authorization/AuthorizationServiceTests.kt

```diff
@@ -270,4 +271,14 @@ class AuthorizationServiceTests {
                 fail("it should have failed as entity type is related to authorization")
             })
     }
+
+    @Test
+    fun `get group membership should return a count at -1 while being no authentification mode`() {
```

Comment:
a count of -1 (not at) / if in no authentication mode (consistency with other tests)